src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/MålgruppeHjelpetekst.tsx

```diff
@@ -21,12 +24,47 @@ export const MålgruppeHjelpetekst = () => (
         </BodyLong>
         <BodyLong size={'small'} spacing>
             Dersom søker oppgir <strong>nedsatt arbeidsevne</strong>, men ikke mottar AAP eller
-            uføretrygd, må du be Nav- kontoret vurdere om vilkårene for nedsatt arbeidsevne i § 11
-            A-3 er oppfylt.
+            uføretrygd, må du be Nav-kontoret vurdere om vilkårene for nedsatt arbeidsevne i §11 A-3
+            er oppfylt.
         </BodyLong>
         <BodyLong size={'small'}>
             Velg <strong>100% sykepenger</strong> hvis søker mottar sykepenger fra en
             fulltidsstilling.
         </BodyLong>
     </ReadMore>
 );
+
+const LæremidlerHjelpetekst = () => (
+    <ReadMore header={'Slik gjør du vurderingen'} size="small">
+        <BodyLong size={'small'} spacing>
+            Dersom søker oppgir å ha <strong>overgangsstønad</strong>, men ikke mottar stønaden, må
+            du be NAY enslig forsørger (kontornummer 4489) vurdere om søker har rett til
+            overgangsstønad. Dersom søker har rett, settes målgruppe overgangsstønad til oppfylt.
+        </BodyLong>
+        <BodyLong size={'small'} spacing>
+            Dersom søker oppgir å ha <strong>omstillingsstønad</strong>, men ikke mottar stønaden,
+            må du be Nav Familie- og Pensjonsytelser vurdere om søker har rett til
+            omstillingsstønad. Dersom søker har rett, settes målgruppe omstillingsstønad til
+            oppfylt. Dersom søker ikke har eller har rett til omstillingsstønad, må du kontrollere
```

Comment:
Hjelpeteksten på Favro har blitt oppdatert
```suggestion
            oppfylt. Dersom søker ikke mottar eller har rett til omstillingsstønad, må du kontrollere
```